.github/workflows/static.yml

```diff
@@ -4,6 +4,9 @@ on: ['push', 'pull_request']
 
 jobs:
   cs:
+    env:
```

Comment:
Just a note, this could be added to a top level `env` key to remove the duplication, but not really necessary. 👍🏻